apps/frontend/src/pages/settings/authorizations.vue

```diff
@@ -13,7 +13,8 @@
 			account. You can manage and review access to your account here at any time.
 		</p>
 		<div v-if="appInfoLookup.length === 0" class="universal-card recessed">
-			You have not authorized any applications.
+			We currently can't display your authorized apps, we're working to fix this. Please visit this
+			page at a later date!
```

Comment:
confused by this message, what's the context behind this?

Reply:
The backend is broken (and has been for ages) - going to write up a linear issue abt it

Essentially the appInfoLookup endpoint doesn't parse the IDs right (broken serde)

Reply:
hmm ok